test/fixtures/dialects/oracle/at_signs.yml

```diff
@@ -3,9 +3,37 @@
 # computed by SQLFluff when running the tests. Please run
 # `python test/generate_parse_fixture_yml.py`  to generate them after adding or
 # altering SQL files.
-_hash: a3e85b165a4d3c6728dddc8257816236fe4bad74db6e1e93316a6f1a4a73e7bb
+_hash: 60284384705beb18d09cd14422636d65e8b4f5af2718a3acf4a99518a08473ba
 file:
-  statement:
+- execute_file_statement:
```

Comment:
I see that a lot of YAML content was added here although the corresponding SQL file didn't change. Is this what you meant about part of the file being parsed incorrectly as a comment? Otherwise, I'm confused. 🙂

Reply:
Yes. They perviously were just treated as comments so didn't appear in the YAMLs. That works as a quick way to exclude non-SQL bits of code (like these execute statements), but means the `@` can't be used elsewhere (like we need in Database links).

Now they are parsed as real statements, hence the YAMLs changed. But it's a good thing for two reasons:
1. Allows `@` symbol to be used elsewhere like we need for Database links
2. Allows these statements to be parse more in future if we want.

So yes this was an intended change.